test-app/tests/unit/macros/t-test.ts

```diff
@@ -1,139 +1,77 @@
 import type { TestContext as BaseTestContext } from '@ember/test-helpers';
-import type { IntlService } from 'ember-intl';
 import { raw, t } from 'ember-intl';
-import { addTranslations, setupIntl } from 'ember-intl/test-support';
+import { addTranslations, setLocale, setupIntl } from 'ember-intl/test-support';
 import { setupTest } from 'ember-qunit';
-import { module, skip, test } from 'qunit';
+import { module, test } from 'qunit';
+
+class ContainerObject extends EmberObject {}
 
 interface TestContext extends BaseTestContext {
-  ContainerObject: typeof EmberObject;
-  intl: IntlService;
-  object: EmberObject;
+  emberObject: EmberObject;
 }
 
-module('Unit | Macros | t', function (hooks) {
+module('Unit | Macro | t', function (hooks) {
   setupTest(hooks);
   setupIntl(hooks, {
-    'no.interpolations': 'text with no interpolations',
+    'no.interpolations': 'Hello',
     'with.interpolations': 'Clicks: {clicks}',
     'with.two.interpolations': 'Coordinates: {x}, {y}',
   });
 
   hooks.beforeEach(function (this: TestContext) {
-    const { owner } = this;
-
-    this.ContainerObject = class extends EmberObject {
-      constructor() {
-        // eslint-disable-next-line prefer-rest-params
-        super(...arguments);
-        setOwner(this, owner);
-      }
-    };
-
-    this.object = this.ContainerObject.extend({
+    this.emberObject = ContainerObject.extend({
       numberClicks: 9,
-      tMacroProperty1: t('no.interpolations'),
-      tMacroProperty2: t('with.interpolations', { clicks: 'numberClicks' }),
-    }).create();
-  });
-
-  test('defines a computed property that translates without interpolations', function (this: TestContext, assert) {
-    assert.strictEqual(
-      this.object.get('tMacroProperty1'),
-      'text with no interpolations',
-    );
-  });
-
-  test('defines a computed property that translates with interpolations', function (this: TestContext, assert) {
-    assert.strictEqual(this.object.get('tMacroProperty2'), 'Clicks: 9');
-  });
-
-  /*
-    FIXME:
+      yCoord: 4,
 
-    This test, even when marked as a TODO (using `todo` from `@ember/test-helpers`),
-    can be picked up by `ember-try` and fail continuous integration.
+      output1: t('no.interpolations'),
 
-    According to @buschtoens, the failure may be due to an intentional change upstream
-    in Ember.js.
+      output2: t('with.interpolations', {
+        clicks: 'numberClicks',
+      }),
 
-    https://github.com/ember-intl/ember-intl/pull/1634/commits/0a39222961cf446b9c8169805cf8e3f56f51976e
+      output3: t('with.two.interpolations', {
+        x: raw(10),
+        y: 'yCoord',
+      }),
+    }).create();
 
-    Consider deleting the test.
-  */
-  skip('allows property to be overridden', function (this: TestContext, assert) {
-    this.object.set('tMacroProperty2', 'A new value');
-    assert.strictEqual(this.object.get('tMacroProperty2'), 'A new value');
+    setOwner(this.emberObject, this.owner);
   });
```

Comment:
This test can no longer be valid, because setting a computed property has been deprecated.

https://deprecations.emberjs.com/v3.x#toc_computed-property-override